meta-lmp-base/recipes-security/optee/optee-client_3.12.0.bb

```diff
@@ -1,6 +1,10 @@
 require optee-client.inc
 
 SRCREV = "347144b79964233e718347e3f0fbe89ab5be46ae"
+
+SRC_URI += " \
+    file://0002-tee-supplicant-disable-RPMB_EMU-by-default.patch \
+"
```

Comment:
We should't need this patch anymore with 3.12, as we're building with RPMB_EMU=0 by default already (optee-client.inc), and for qemu we got the following:

```
$ cat meta-lmp-bsp/recipes-security/optee/optee-client_3.12.0.bbappend
# Enable RPMB emulation on qemuarm64 to easy testing
EXTRA_OEMAKE_append_qemuarm64 = " \
    RPMB_EMU=1 \
"
```

Reply:
@ricardosalveti ok, let me just drop that patch